src/gui/sdlmain.cpp

```diff
@@ -2581,6 +2600,7 @@ bool GFX_Events()
 				                  event.window.data2);
 				continue;
 
+			case SDL_WINDOWEVENT_FOCUS_GAINED: FALLTHROUGH;
```

Comment:
Remove this FALLTHROUGH, please. FALLTHROUGH is required only in places where it's not trivial (some code is placed inside the case branch and break statement is missing).

Reply:
Got it; thanks for the explanation of when to use FALLTHROUGH!

Reply:
Ah - this will also avoid the need to update the warnings limit too :+1: